.run/App.Server.run.xml

```diff
@@ -16,7 +16,7 @@
     <option name="PROJECT_WORKING_DIRECTORY_TRACKING" value="1" />
     <option name="PROJECT_KIND" value="DotNetCore" />
     <option name="PROJECT_TFM" value="net6.0" />
-    <browser url="https://localhost:7081/chat/the-actual-one" name="98ca6316-2f89-46d9-a9e5-fa9e2b0625b3" start="true" />
+    <browser url="https://local.actual.chat/chat/the-actual-one" name="98ca6316-2f89-46d9-a9e5-fa9e2b0625b3" />
```

Comment:
why are we trying to replace localhost with local.actual.chat? does local.actual.chat support page autoreload on connection restore?

Reply:
`local.actual.chat` uses nginx so image proxy works locally as well. Additionally notifications also works when the site is not localhost

Reply:
Why do you expect it doesn't reload on conneciton restore?

Reply:
Nginx will show you 502 error if local dev host is stopped. 502 is not automatically reloaded when host is restarted.

Anyways, I think it's such a minor inconvenience comparing to benefits you mentioned above